### PR TITLE
Pass tmpdir chmod arg to other Bowtie2 call

### PR DESCRIPTION
**Pull request by terrycojones**
And some small playing with DIAMOND / SAM output (still not working).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 4.0.0 DATE......................................................................
```
Comment (bamueh) [inline CHANGELOG.md:1]: Shouldn't this have a date?

File: dark/diamond/sam.py [hunk 2]
```diff
@@ -160,15 +159,17 @@ def _SAMLine(self, match, protein, genome):
...
         # queryStartInGenome = matchStartInGenome - match['qstart'] - 1
         # orientations = ranges.orientations()
 
-        print('ranges', ranges)
-        print('protein', protein)
-        print('match', match)
+        # print('ranges', ranges)
+        # print('protein', protein)
+        # print('match', match)
+
```
Comment (bamueh) [inline dark/diamond/sam.py:172]: Can't ^^ lines just go?

File: CHANGELOG.md (hunk 1 above)
Comment (terrycojones) [reply CHANGELOG.md:1]: Shit - thanks! I put things in there so I can't miss them when I go to make a pull request. But, I missed it :-(

File: dark/diamond/sam.py (hunk 2 above)
Comment (terrycojones) [reply dark/diamond/sam.py:172]: I want to leave them in as that code isn't finished. There are some `raise NotImplementedError` lines in there as well.

**Pull request merged**